client/lib/reader-lists-feeds/constants.js

```diff
@@ -0,0 +1,6 @@
+module.exports.action = {
```

Comment:
let's use the new export syntax here, aka
```
export const RECEIVE_READER_LIST_FEEDS = 'RECEIVE_READER_LIST_FEEDS';
```

By making these const, the compiler can catch cases where we try to modify them